conbench/__init__.py

```diff
@@ -14,7 +15,15 @@ def create_application(config):
 
     application = f.Flask(__name__)
     application.config.from_object(config)
+
     _init_application(application)
+
+    if application.config["DEBUG"]:
+        c = json.dumps(application.config, sort_keys=True, default=str, indent=2)
```

Comment:
I don't think I knew that `json.dumps` had such convenience bits as sorting and indents ❤️

Reply:
yeah! :) True magic here is the `default=str` which is a meaningful fallback for objects that are not JSON-serializable, such as `datetime.timedelta` objects (needed that here).